README.md

```diff
@@ -1,23 +1,15 @@
 # echoshader
 
-Core ocean sonar data visualization functionalities based on the HoloViz suite of tools.
+Core ocean sonar data visualization functionalities based on the [HoloViz](https://holoviz.org/) suite of Python tools.
```

Comment:
I have mixed thoughts on the word "functionalities" and "core". What do you think of the text below instead?:

"Open Source Python package for building ocean sonar data visualizations based on the [HoloViz](https://holoviz.org/) suite of Python tools."?